contents/blog/how-we-improved-feature-flags-resiliency.md

```diff
@@ -0,0 +1,64 @@
+---
+date: 2021-08-20
+title: How we improved feature flag resiliency
```

Comment:
How we made feature flags faster and more reliable

Reply:
I like faster and more reliable better as a title